source/app.py

```diff
@@ -0,0 +1,2 @@
+if __name__ == "__main__":
+    print('hello world!')
```

Comment:
Can you change this to logging instead of print statement
```suggestion
    logger.info('hello world!')
```